### PR TITLE
SAK-40373: Appears name in 'Assignment List by Student' in assignment…

### PR DESCRIPTION
**Pull request by jesusmmp**
…s without submission

### CODE REVIEW

File: assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_student_list_submissions.vm [hunk 1]
```diff
@@ -173,7 +173,9 @@
 													#set ($submissionSubmitter = $!service.getSubmissionSubmittee($submission))
 													#if ($!submissionSubmitter.isPresent())
 														#set ($submitterId = $!submissionSubmitter.get().getSubmitter())
-														<br />$tlang.getString("listsub.submitted.by") $studentMembersMap.get($submitterId).getDisplayName()
+														#if ($!submissionSubmitter.get().getSubmittee())
```
Comment (ern) [inline assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_student_list_submissions.vm]: If the submitter is not in the map this is probably some other issue going on... Maybe adding some an else indicating that would be ideal?

Comment (ottenhoff) [reply assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_student_list_submissions.vm]: @jesusmmp can we add an "else"?

Comment (ern) [reply assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_student_list_submissions.vm]: I left comments on the JIRA but I could not replicate this on master, can you please provide the steps.

**Comment by ern** [comment]
Closing for now feel free to reopen when the issue is can be replicated.

**Pull request closed without merging**